lib/miq_automation_engine/engine/miq_ae_object.rb

```diff
@@ -245,6 +245,16 @@ def process_assertions(message)
       process_filtered_fields(['assertion'], message)
     end
 
+    def user_info_attributes(user)
+      objects_hash = {'user'      => user,
+                      'tenant'    => user.current_tenant,
+                      'miq_group' => user.current_group}
+      objects_hash.each do |k, v|
```

Comment:
@mkanoor No need to create a temporary object here if you are not going to reference it once.  Just setup the hash and call `.each` on it.